op-batcher/batcher/channel_manager.go

```diff
@@ -260,6 +260,10 @@ func (s *channelManager) processBlocks() error {
 		if s.currentChannel.IsFull() {
 			break
 		}
+		if s.rollupCfg.KromaMPTTime != nil && block.Time() == *s.rollupCfg.KromaMPTTime-s.rollupCfg.BlockTime {
+			s.currentChannel.channelBuilder.setFullErr(errors.New("reached the block just before KromaMPTTime"))
```

Comment:
It's nit, but how about setting timeout instead of fullErr? It does not change the result, but seems like timeout is more natural for me.

Reply:
That's a good suggestion. However, since they are used with the l1 block number as input, it could cause some confusion and added complexity. Also, since these are based on fullErr after all, I think the current implementation is better

Reply:
Agreed, then let's go with fullErr.